04-spatial-operations.qmd

```diff
@@ -494,12 +495,46 @@ Terrain processing, the calculation of topographic characteristics such as slope
 * `planform_curvature` 	Zevenbergen and Thorne (1987) doi: 10.1002/esp.3290120107
 * `profile_curvature` 	Zevenbergen and Thorne (1987) doi: 10.1002/esp.3290120107
 
+### Zonal operations
 
+Just like focal operations, zonal operations apply an aggregation function to multiple raster cells. However, a second raster, usually with categorical values, defines the zonal filters (or 'zones') in the case of zonal operations, as opposed to a predefined neighborhood window in the case of focal operation presented in the previous section. Consequently, raster cells defining the zonal filter do not necessarily have to be neighbors. Our grain size raster is a good example, as illustrated in the right panel of Figure 3.2: different grain sizes are spread irregularly throughout the raster. Finally, the result of a zonal operation is a summary table grouped by zone which is why this operation is also known as zonal statistics in the GIS world. This is in contrast to focal operations which return a raster object.
```

Comment:
I think our style is sentence based breaks, so each sentence is on a new line, e.g.:

```
Just like focal operations, zonal operations apply an aggregation function to multiple raster cells.
However, a second raster, usually with categorical values, defines the zonal filters (or 'zones') in the case of zonal operations, as opposed to a predefined neighborhood window in the case of focal operation presented in the previous section.
Consequently, raster cells defining the zonal filter do not necessarily have to be neighbors.
Our grain size raster is a good example, as illustrated in the right panel of Figure 3.2: different grain sizes are spread irregularly throughout the raster.
Finally, the result of a zonal operation is a summary table grouped by zone which is why this operation is also known as zonal statistics in the GIS world.
This is in contrast to focal operations which return a raster object.
```

Reply:
Sure, thanks!